.circleci/config.yml

```diff
@@ -102,7 +102,7 @@ jobs:
       - run: |
           PR_NUM=$(echo "$CIRCLE_PULL_REQUEST" | grep -o "[0-9]*")
           if [ -z "$PR_NUM" ]; then
-            DIFFBRANCH="origin/staging"
+            DIFFBRANCH="ros2"
```

Comment:
I'm not sure that this is the right default behavior here. Should we even be running tidy/pretty-diff on regular branch builds?